db/migrate/20150427224114_add_o_auth_to_app.rb

```diff
@@ -0,0 +1,9 @@
+class AddOAuthToApp < ActiveRecord::Migration
+  def change
+    add_column :apps, :redirect_uri, :string
+    add_column :apps, :homepage, :string
+    add_column :apps, :description, :string
+    add_column :apps, :privileged, :boolean, null: false, default: false
```

Comment:
Wouldn't it be better to change to something like "access_status" and use a string field, so we can use it for more statuses? (something like silver/gold app status, each one with more permissions)

I don't know if I'm optimizing prematurely, but I think it's better to ask this on a PR rather than having to alter the table in the future